Tests/DeepLearningTests/ComplexAutoDiffTests.swift

```diff
@@ -0,0 +1,98 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+//
+// Complex API tests.
+
+// TODO: remove import
+import TensorFlow
+
+import StdlibUnittest
```

Comment:
We should not use `StdlibUnittest` in swift-apis since it's a SwiftPM package. Could you please refactor this to use XCTest like other test files in this directory?

And could you please merge the two test files into one?

Reply:
Yes I noticed the error when I was looking as to why the tests were failing. Let me update the tests so it's in one file!